src/gapstate.h

```diff
@@ -123,6 +123,9 @@ typedef struct GAPState {
     Obj  BaseShellContext;
     Obj  ErrorLVars;        // ErrorLVars as modified by DownEnv / UpEnv
     Int  ErrorLLevel;       // record where on the stack ErrorLVars is relative to the top, i.e. BaseShellContext
+    void (*JumpToCatchFunc)(); // Callback that is called in FuncCALL_TO_CATCH,
```

Comment:
It is called in `FuncJUMP_TO_CATCH`, not CALL_...

Would you mind renaming the callback to `JumpToCatchCallback`? I find that slightly more helpful when reading the code. But it's probably subjective, so *shrug*